lib-object-oriented/parser.js

```diff
@@ -10,24 +10,82 @@ export default class Parser {
   getTokens() {
     const lexer = new Lexer( this.content )
     this.tokens = lexer.lex()
-    }
+  }
 
   parse() {
```

Comment:
I refuse to review a function over 20 lines of code.  My feedback is to clean this up.  :)